app/views/layouts/user_mailer.html.slim

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+html xmlns="http://www.w3.org/1999/xhtml" 
+  head
+    = stylesheet_link_tag "email/email"
+    meta content="text/html; charset=utf-8" http-equiv="Content-Type"
+    meta content="width=device-width" name="viewport"
+    title
+      = message.subject
+  body
+    table.body data-made-with-foundation=""
+      tr
+        td.center align="center" valign="top" 
+          center data-parsed="" 
+            table.spacer.float-center
+              tbody
+                tr
+                  td height="10px" style="font-size:10px;line-height:10px;"   
+            table.container.float-center
+              tbody
+                tr
+                  td
+                    table.spacer
+                      tbody
+                        tr
+                          td height="35px" style="font-size:35px;line-height:35px;"   
+                    table.row
+                      tbody
+                        tr
+                          th.align-middle.small-12.large-12.columns.first.last
+                            table
+                              tr
+                                th
+                                  = image_tag("logo.png", class: "small-float-center")
```

Comment:
SVG support in various email clients is still spotty, so using a PNG is likely the most reliable method.  Reference: https://css-tricks.com/a-guide-on-svg-support-in-email/